web/src/app/alerts/components/AlertDetailLogs.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import p from 'prop-types'
+import Divider from '@material-ui/core/Divider'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Button from '@material-ui/core/Button'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import { DateTime, Interval } from 'luxon'
+import { POLL_INTERVAL } from '../../config'
+
+const LIMIT = 149
+
+const query = gql`
+  query getAlert($id: Int!, $input: AlertRecentEventsOptions) {
+    alert(id: $id) {
+      data: recentEvents(input: $input) {
+        nodes {
+          timestamp
+          message
+        }
+        pageInfo {
+          hasNextPage
+          endCursor
+        }
+      }
+    }
+  }
+`
+export default function AlertDetailLogs(props) {
+  const [poll, setPoll] = useState(POLL_INTERVAL)
+  const { data, error, loading, fetchMore } = useQuery(query, {
+    pollInterval: poll,
+    variables: { id: props.alertID, input: { after: '', limit: 35 } },
+  })
+
```

Comment:
Let's create two variables that we can use later on to make it easier to have our conditional rendering checks in place:

```
const events = _.get(data, 'alert.data.nodes', [])
const pageInfo = _.get(data, 'alert.data.pageInfo', {})
```